apps/explorer/src/components/NFTThumbnail.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { useQuery } from '@tanstack/react-query'
+import { memo } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { useTheme } from 'styled-components'
+
+import { queries } from '@/api'
+import LoadingSpinner from '@/components/LoadingSpinner'
+import VideoThumbnail from '@/components/VideoThumbnail'
+
+interface NFTThumbnailProps {
+  src: string
+  size?: number
+  border?: boolean
+  borderRadius?: number
+  autoPlay?: boolean
+  playOnHover?: boolean
+  showPlayIconIfVideo?: boolean
+}
+
+const NFTThumbnail = memo(
+  ({ src, size, border, borderRadius, autoPlay, showPlayIconIfVideo, playOnHover }: NFTThumbnailProps) => {
+    const { t } = useTranslation()
+    const theme = useTheme()
+
+    const { data: dataType, isLoading: isDataTypeLoading } = useQuery({
+      ...queries.assets.NFTsData.type(src || ''),
+      enabled: !!src
+    })
```

Comment:
Maybe an issue so that we don't forget about it will be useful (ie _"Use skipToken instead of enabled in explorer"_). I'll resolve the discussion then.

Reply:
#728